docs/network/smart-rollup-nodes.md

```diff
@@ -83,4 +85,58 @@ For example, this query gets the health of the node:
    curl -s http://localhost:8932/health
    ```
 
-Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](/network/evm-nodes).
+Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](./evm-nodes).
+
+## Running the Smart Rollup node from Etherlink genesis
+
+Running the Smart Rollup node from Etherlink genesis takes time but does not rely on snapshots created by third parties.
+
+For simplicity, these steps show how to run the Smart Rollup node in observer mode:
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
+The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/tezos/tezos/-/releases.
+
+1. Initialize the local context of the node, which is where it stores local data:
+
+   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   The default value is `$HOME/.tezos-smart-rollup-node`.
+   1. Initialize the local context by running this command:
+
+      ```bash
+      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+        with operators --data-dir $sr_observer_data_dir \
+        --pre-images-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0
+      ```
+
+      This command generates a minimal configuration file (`config.json`) in the local data folder:
+
+      ```json
+      { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+        "smart-rollup-node-operator": {}, "fee-parameters": {}, "mode": "observer",
+        "pre-images-endpoint":
+          "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0" }
+      ```
+
+      This configuration uses the preimages that the Tezos Foundation hosts on a file server on a so-called "preimages endpoint".
+      To build the preimages yourself, see [Building the Etherlink kernel](./building-kernel).
+
+1. Start the Smart Rollup node in observer mode by running this command and using the RPC endpoint of a layer 1 node that is running in archive mode:
+
+   ```bash
+   octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
+     --data-dir $sr_observer_data_dir
+   ```
+
+   The node must be running in archive mode to provide your Smart Rollup node complete information about the Etherlink state from genesis.
```

Comment:
Nit: 
A rolling or full node contains information for only the last couple of cycle, but for theses cycles the information is complete. Maybe my proposition is not the best.
 
```suggestion
   The layer 1 node must be running in archive mode to provide your Smart Rollup node  information about the Etherlink state since its genesis.
```